Networking/Networking/Mapper/BlazeImpressionsMapper.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// Mapper: BlazeImpressions
+///
+struct BlazeImpressionsMapper: Mapper {
+
+    /// (Attempts) to convert a dictionary into a BlazeImpressions entity
+    ///
+    func map(response: Data) throws -> BlazeImpressions {
+        let decoder = JSONDecoder()
+        return try decoder.decode(BlazeImpressions.self, from: response)
+    }
```

Comment:
We can use `convertFromSnakeCase` as `keyDecodingStrategy` to avoid adding a custom decoder and coding keys in `BlazeImpressions`. 

```suggestion
    func map(response: Data) throws -> BlazeImpressions {
        let decoder = JSONDecoder()
        decoder.keyDecodingStrategy = .convertFromSnakeCase
        return try decoder.decode(BlazeImpressions.self, from: response)
    }
```

Reply:
This is very handy. Nice!